.github/ISSUE_TEMPLATE/existing_data_updates.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Integrate New Year of Data
+about: Check-list for integrating a new year of data
+title: ''
+labels: new-data
+assignees: ''
+
+---
+
+### New year of data integration check-list:
+
+Based on the [Existing Data Updates Docs](https://catalystcoop-pudl.readthedocs.io/en/dev/dev/existing_data_updates.html)
```

Comment:
This whole template is showing up as deleted and then re-added I think because the percentage content change is large enough that git doesn't recognize it as the same file? I tried to prevent this by running seperate commits - one for the file changes and one for the file rename - but that didn't seem to help.

https://stackoverflow.com/questions/7938582/how-does-git-detect-similar-files-for-its-rename-detection

Reply:
that's annoying but of well :-/